plugins/json-multisheet-extractor/README.md

```diff
@@ -0,0 +1,78 @@
+<!-- START_INFOCARD -->
+
+The `@flatfile/json-multisheet-extractor` plugin parses a JSON file and extracts second-level nested objects into first level defined Sheets in Flatfile.
+
+**Event Type:** 
+`listener.on('file:created')`
+
+**Supported file types:** 
+`.json`
+
+<!-- END_INFOCARD -->
+
+> When embedding Flatfile, this plugin should be deployed in a server-side listener. [Learn more](/docs/orchestration/listeners#listener-types)
+
+
+
+## Parameters
+
+#### `options.chunkSize` - `default: "10_000"` - `number` - (optional)
```

Comment:
AI bot suggestion on format https://github.com/FlatFilers/flatfile-plugins/pull/693/files#r1826003560

I took some of it's suggestions, but didn't want to deviate too far from the pattern in this repo